L/libgsd/build_tarballs.jl

```diff
@@ -26,6 +28,7 @@ platforms = filter!(!Sys.iswindows, platforms)
 
 # The products that we will ensure are always built
 products = [
+    FileProduct("share/gsd.h", :gsd_h), # header file is needed for the use of CBinding.jl in follow up wrapper of GSD.jl
```

Comment:
```suggestion
    FileProduct("include/gsd.h", :gsd_h),
```

Reply:
@ywitzky I had removed the comment because it isn't necessary. This package is a library, including header files is a normal thing to do in a library and doesn't need an explanation.

Reply:
@imciner2  Ok thanks. That makes sense. Ill fix this.